migrations/sqlite3/20240925160534_fix_symbol_length2.sql

```diff
@@ -0,0 +1,10 @@
+-- +up
+-- +begin
+SELECT 1;
```

Comment:
This migration file could be dropped because nothing is changed?

Reply:
sqlite does not support column modification